glide.lock

```diff
@@ -194,24 +190,32 @@ imports:
 - name: github.com/elazarl/go-bindata-assetfs
   version: 30f82fa23fd844bd5bb1e5f216db87fd77b5eb43
 - name: github.com/emicklei/go-restful
-  version: 892402ba11a2e2fd5e1295dd633481f27365f14d
+  version: 89ef8af493ab468a45a42bb0d89a06fccdd2fb22
   subpackages:
   - log
   - swagger
 - name: github.com/fatih/color
   version: 9131ab34cf20d2f6d83fdc67168a5430d1c7dc23
 - name: github.com/gambol99/go-marathon
-  version: 15ea23e360abb8b25071e677aed344f31838e403
```

Comment:
Not very sure why this shows up as a diff, the `glide.yaml` has `d672c6fbb499596869d95146a26e7d0746c06c54` version for `github.com/gambol99/go-marathon` which came from a prior [commit](https://github.com/containous/traefik/commit/4e238280bceae43011b66613b94bd9f4ac1c233c) (https://github.com/containous/traefik/blame/master/glide.yaml#L96)